src/bucket.ts

```diff
@@ -1644,7 +1653,8 @@ class Bucket extends ServiceObject {
         },
         (err, resp) => {
           if (err) {
-            callback(err, null, null, resp);
+            // tslint:disable-next-line:no-any
+            (callback as any)(err, null, null, resp);
```

Comment:
This is a sad hack ☹️I suppose we can fix that later with how we call upon the paginator

Reply:
Lol yeah, it ain't pretty.  First pass, I want to get all the types we know are missing in.  We can come back and clean up the weird `any` stuff laster.  I don't mind as much here since this is internal.